test/integration-test.sh

```diff
@@ -106,8 +106,17 @@ run_command "$API1" "send ,$HOPR_ADDRESS2 'hello, world'"
 echo "- Node 1 open channel to Node 2"
 run_command "$API1" "open $HOPR_ADDRESS2 0.1"
 
-echo "- Node 1 send 1 hop message to self via node 2"
-run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world'"
-
-echo "- Node 2 should now have a ticket"
+echo "- Node 1 send 10x 1 hop message to self via node 2"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 1'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 2'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 3'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 4'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 5'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 6'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 7'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 8'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 9'"
+run_command "$API1" "send $HOPR_ADDRESS2,$HOPR_ADDRESS1 'hello, world 10'"
+
+echo "- Node 2 should now likely have a ticket"
```

Comment:
It is probably worth to wait ~1 sec

Reply:
I don't believe in hacky timeouts in tests.